x-pack/plugins/spaces/server/lib/copy_to_spaces/copy_to_spaces.test.ts

```diff
@@ -222,7 +222,6 @@ describe('copySavedObjectsToSpaces', () => {
               "_maxListeners": undefined,
               "_read": [Function],
               "_readableState": ReadableState {
-                "autoDestroy": false,
```

Comment:
I'm not sure how/why you were prompted to change this snapshot. When I tested it locally, this modified snapshot failed.

<details><summary>Test results</summary>

```bash
 FAIL  plugins/spaces/server/lib/copy_to_spaces/copy_to_spaces.test.ts
  copySavedObjectsToSpaces
    ✕ uses the Saved Objects Service to perform an export followed by a series of imports (23ms)
    ✓ doesn't stop copy if some spaces fail (5ms)
    ✓ handles stream read errors (10ms)

  ● copySavedObjectsToSpaces › uses the Saved Objects Service to perform an export followed by a series of imports

    expect(received).toMatchInlineSnapshot(snapshot)

    Snapshot name: `copySavedObjectsToSpaces uses the Saved Objects Service to perform an export followed by a series of imports 3`

    - Snapshot
    + Received

    @@ -12,10 +12,11 @@
              },
              "_eventsCount": 3,
              "_maxListeners": undefined,
              "_read": [Function],
              "_readableState": ReadableState {
    +           "autoDestroy": false,
                "awaitDrain": 0,
                "buffer": BufferList {
                  "head": null,
                  "length": 0,
                  "tail": null,
    @@ -76,10 +77,11 @@
              },
              "_eventsCount": 3,
              "_maxListeners": undefined,
              "_read": [Function],
              "_readableState": ReadableState {
    +           "autoDestroy": false,
                "awaitDrain": 0,
                "buffer": BufferList {
                  "head": null,
                  "length": 0,
                  "tail": null,

      206 |     `);
      207 | 
    > 208 |     expect((importSavedObjectsFromStream as jest.Mock).mock.calls).toMatchInlineSnapshot(`
          |                                                                    ^
      209 |       Array [
      210 |         Array [
      211 |           Object {

      at Object.it (plugins/spaces/server/lib/copy_to_spaces/copy_to_spaces.test.ts:208:68)

 › 1 snapshot failed.
Snapshot Summary
 › 1 snapshot failed from 1 test suite. Inspect your code changes or re-run jest with `-u` to update them.

Test Suites: 1 failed, 1 total
Tests:       1 failed, 2 passed, 3 total
Snapshots:   1 failed, 4 passed, 5 total
Time:        3.258s
Ran all test suites matching /x-pack\/plugins\/spaces\/server\/lib\/copy_to_spaces\/copy_to_spaces.test.ts/i.
```

</details>

Reply:
Turns out I was using a slightly different version of node, I've reverted this change.